contrib/node/src/python/pants/contrib/node/subsystems/BUILD

```diff
@@ -2,6 +2,12 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
 
 python_library(
+  sources=[
```

Comment:
Why enumerate the sources explicitly?

Reply:
It's an old habit. In my previous projects, it was recommended to explicitly list all sources and strongly discouraged usage of glob/rglob or equivalent.
I don't know what the Pants norm is.  Do we have a recommendation on this topic?

Reply:
I think the less boilerplate the better. We don't list sources unless we have to. They default to a glob over the appropriate file suffix, which is almost always what you want anyway.

Reply:
done.